plugins/smart_proxy_dns_powerdns/copyright

```diff
@@ -0,0 +1,28 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: smart_proxy_dns_powerdns
+Source: https://github.com/theforeman/smart_proxy_dns_powerdns
+
+Files: *
+Copyright: 2015 Red Hat Inc. <foreman-dev@googlegroups.com>
```

Comment:
Tbh I didn't really think of a licence and don't know what the policy is. I'm fine with just my name there but I don't care that much as long as the plugin is used and open :)